.jenkins/actions/run_standalone.sh

```diff
@@ -60,6 +60,9 @@ fi
 if [ ! -d "${BENCHMARK_DIR}" ] ; then
     exitError 1005 ${LINENO} "Benchmark directory ${BENCHMARK_DIR} does not exist"
 fi
+if [ "${SAVE_CACHE}" == "true" ] ; then
+    TIMESTEPS=2
```

Comment:
should we do 0 here?

Reply:
Not clear. For the time being I think we should leave it as is until we compile stencils via initialization and we have to change the logic here anyways.